pkg/errors/factory.go

```diff
@@ -50,22 +71,11 @@ func GetError(err error) error {
 	switch {
 	case isEmptyConfigError(err):
 		return newNoKubeConfig(err.Error())
+	case isStatusError(err):
```

Comment:
Should this be moved up and considered first?

Reply:
+1
I'd say that `no configuration has been provided` message is quiet distinctive from k8s api erros, but it's good idea to have it at first place to check.